app/carriers/labels_carrier.rb

```diff
@@ -1,6 +1,11 @@
 class LabelsCarrier
+  attr_reader :label_status
+
+  LABEL_STATUS = %w(backlog ready progress verify codereview)
+
   def initialize(events)
     @events = events
+    @label_status = LABEL_STATUS
```

Comment:
No need to use instance variable, please use `LabelsCarrier::LABEL_STATUS` in views.